src/complex/Utilities/StringUtilities.hpp

```diff
@@ -212,6 +212,10 @@ template <typename T>
 inline std::string number(T arg)
 {
   std::stringstream ss;
+  if(std::is_floating_point_v<T>)
```

Comment:
```suggestion
  if constexpr(std::is_floating_point_v<T>)
```
`if constexpr` will make sure this is only generated for floating point types.